tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -0,0 +1,353 @@
+  It is the first page that users would look at in order to under
+  the overall performance of their TPU workloads. The page has five
+  sections of results:
+     (1) Performance summary
+     (2) Step-time graph
+     (3) Top 10 TensorFlow Operations executed on TPU
+     (4) Run environment
+     (5) Recommendation for next steps
+-->
+
+<link rel="import" href="../vz-line-chart/vz-line-chart.html">
+<link rel="import" href="../paper-card/paper-card.html">
+
+<dom-module id="overview-page">
+  <template>
+    <style>
+     paper-card {
+       margin: 5px;
+       --paper-card-header-color: white;
+       --paper-card-header: background-color: blue;
+     }
+     .steptime-average {
+       font-weight:bold;
+       font-style:italic;
+       color:red;
+     }
+     .table-style {
+       table-layout:auto;
+       width:95%;
+     }
+     .top-ops-table {
+       width:100%;
+     }
+     .x-axis-title {
+       width:80%;
+       text-align:center;
+       text-transform: capitalize;
+     }
+     .y-axis-title {
+       width:10px;
+       writing-mode:bt-rl;
+       transform: rotate(270deg);
+       text-align:center;
+       white-space: nowrap;
+       text-transform: capitalize;
+     }
+     div.bottleneck-statement{
+       color:red;
+       font-weight:bolder;
+       font-style:italic;
+     }
+     div.errorMessage {
+       color:red;
+       font-size:0.5cm;
+       font-weight: bolder;
+       margin-top:0.5cm;
+       margin-bottom:0.5cm;
+     }
+     div.bottleneckTips{
+       color:#000000;
+     }
+     div.bottleneckTips a{
+       color:#ff33cc;
+       text-decoration:underline;
+     }
+     div.documentationTips {
+       color:#666699;
+     }
+     div.documentationTips a{
+       color:#33cc00;
+       text-decoration:underline;
+     }
+    </style>
+    <div class="errorMessage" hidden$="[[!_show_error_message]]"> <span>[[_error_message]]</span></div>
+    <div class="while_page_content" hidden$="[[_show_error_message]]">
+      <div class="horizontal layout">
+	<paper-card heading="Performance Summary">
+	  <div class="card-content">
+	    <p><b>Average step time</b> (lower is better): <b><span class="steptime-average" >[[_steptime_ms_average]] ms</span> </b> <i style="opacity:0.7">(standard deviation = <span>[[_steptime_ms_stddev]]</span> ms)</i></p>
+	    <p><b>Host idle time</b> (lower is better): <span>[[_host_idle_time_percent]]</span></p>
+	    <p><b>TPU idle time</b> (lower is better): <span>[[_device_idle_time_percent]]</p>
+	      <p><b>Utilization of TPU Matrix Units</b> (higher is better): <span>[[_mxu_utilization_percent]]</span></p>
+	  </div>
+	</paper-card>
+	<paper-card heading="Step-time Graph" >
+	  <div class="card-content">
+	    <table class="table-style">
+	      <tr>
+		<td><div class="y-axis-title">milliseconds</div></td>
+		<td>
+		  <vz-line-chart id="device_step_chart" style="width:860px; height:200px"></vz-line-chart>
```

Comment:
Could these dimensions be specified in embedded (instead of inline styles) to separate design from content?

Reply:
Fixed.